docs/pages/index.rst

```diff
@@ -9,26 +9,74 @@ When rendering a scene from the Matterport3D dataset, *Habitat-Sim* achieves
 several thousand frames per second (FPS) running single-threaded, and reaches
 over 10,000 FPS multi-process on a single GPU!
 
-`Tutorials`_
-============
+The Habitat platform includes *Habitat-Sim* and `Habitat-Lab <http://aihabitat.org/docs/habitat-lab/>`_. To learn how these fit together, see our `ECCV 2020 tutorial series <https://aihabitat.org/tutorial/2020/>`_.
```

Comment:
This is awkward. It's hard to properly introduce a user to the overall Habitat framework when this page is technically only for Habitat-Sim. Maybe Hab-Sim and Hab-Lab will be unified in the glorious Hab 3.0 future, and we can unify the docs on that day, too.

Reply:
Back then we had different repos because of several reasons:
- Sim was expected to be C++ only, while Lab Python
- Users may need Sim only if they aren't from academia or EmbodiedAI domain
- Dependencies: Habitat Lab was up to have a lot of ML based requirements like PyTorch, TensorBoard that are totally not required by Sim users